cli/embedded-app/src/main.ts

```diff
@@ -1,11 +1,17 @@
+// Required polyfills for iframe-coordinator IE11 support
+import 'core-js/stable';
+import 'custom-event-polyfill/polyfill';
+import '@webcomponents/custom-elements/src/native-shim';
+import '@webcomponents/custom-elements';
+// IE11 compatible iframe-coordinator import
```

Comment:
Could we add a top shim layer that auto picked the right dist?  I guess that would make it a bit bloaty for apps that wouldn't need the es5 version...

Reply:
Yeah, we'd have to include both version of the lib in the app build, which would be overkill, IMO.